Mathlib/Combinatorics/SimpleGraph/DegreeSum.lean

```diff
@@ -113,6 +108,21 @@ theorem sum_degrees_eq_twice_card_edges : ∑ v, G.degree v = 2 * G.edgeFinset.c
   G.dart_card_eq_sum_degrees.symm.trans G.dart_card_eq_twice_card_edges
 #align simple_graph.sum_degrees_eq_twice_card_edges SimpleGraph.sum_degrees_eq_twice_card_edges
 
+/-- The complete graph on `n` vertices has `n.choose 2` edges. -/
+theorem top_card_edges_eq_card_choose_two :
+    (⊤ : SimpleGraph V).edgeFinset.card = (Fintype.card V).choose 2 := by
```

Comment:
I think we should rather have this lemma
```suggestion
    (⊤ : SimpleGraph V).edgeFinset = {e | ¬ e.IsDiag} := by
```

Reply:
Surely you mean
```suggestion
    (⊤ : SimpleGraph V).edgeFinset = Finset.univ.filter fun e => ¬e
    IsDiag := by
```

Reply:
Done (for both variations).

Reply:
I would hope the proof now follows in terms of some general lemma about the cardinality of `Sym2`

Reply:
Done.